### PR TITLE
Move zip and json to data folder

### PR DESCRIPTION
**Pull request by werdnaH**
@kjlubick @jonathanmetzman

### CODE REVIEW

**Comment by kjlubick** [comment]
This change is not what we want. What was requested was that the zip files go in $SRC/skia_data instead of $SRC/skia

File: projects/skia/Dockerfile [hunk 1]
```diff
@@ -34,56 +34,58 @@ WORKDIR $SRC/skia/third_party/externals/swiftshader/
 # TODO(metzman): Come up with a better long term solution, such as downloading
 # prebuilt libraries, than pinning swiftshader to a known working revision.
 RUN git checkout 45510ad8a77862c1ce2e33f0efed41544f5f048b
-WORKDIR $SRC/skia
+WORKDIR $SRC/skia_data
```
Comment (kjlubick) [inline projects/skia/Dockerfile:37]: I don't think you want to change the work dir.

Comment (jonathanmetzman) [reply projects/skia/Dockerfile:37]: Yeah I would just undo that WORKDIR change.
Then I can land this.

**Comment by jonathanmetzman** [comment]
I'll land this after CI passes. I made the last change myself.

**Pull request merged**